apps/gaia/src/db/index.ts

```diff
@@ -203,11 +204,36 @@ export const updateObservation = async (id, observation: string) => {
   // language=PostgreSQL
   const result = await query(
     `
-              update observations
-              set long_desc = $1
-              where id = $2
-    `,
+                update observations
+                set long_desc = $1
+                where id = $2
+        `,
     [observation, id]
   )
   return result.rowCount
 }
+
+export const insertImage = async (
+  imageId,
+  objectKey,
+  schoolId,
+  studentId: string
+) => {
+  // language=PostgreSQL
+  await query(
```

Comment:
I think we could combine these 2 queries into a single call that uses transaction for better performance and data integrity.